LICENSE

```diff
@@ -200,9 +200,9 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 =======================================================================
-Apache DolphinScheduler (incubating) Subcomponents:
+Apache DolphinScheduler  Subcomponents:
```

Comment:
I have checked and there is no problem.
It would be better if one extra space could be removed.